awx/main/credential_plugins/tss.py

```diff
@@ -54,7 +54,7 @@
 
 def tss_backend(**kwargs):
     if kwargs.get("domain"):
-        authorizer = DomainPasswordGrantAuthorizer(kwargs['server_url'], kwargs['username'], kwargs['password'], kwargs['domain'])
+        authorizer = DomainPasswordGrantAuthorizer(kwargs['server_url'], kwargs['username'], kwargs['domain'], kwargs['password'])
```

Comment:
just to be safe we can use keyword arguments instead of positional
```suggestion
        authorizer = DomainPasswordGrantAuthorizer(base_url=kwargs['server_url'], username=kwargs['username'], domain=kwargs['domain'], password=kwargs['password'])
```